test/org/redhatcop/util/files/params/notifications-build.params

```diff
@@ -0,0 +1,5 @@
+APPLICATION_NAME=notifications-test
+NAMESPACE=pipeline-library-test
+SOURCE_REPOSITORY_URL=https://github.com/etsauer/pipeline-library.git
+SOURCE_REPOSITORY_REF=shared-lib
+CONTEXT_DIR=test/org/redhatcop/util/
```

Comment:
Want to set this as the default in the template and remove here? I'm fine either way but prefer consistency ;)

Reply:
I think on this one, I would prefer to leave the path blank. Reasoning is, once we have multiple packages in this repo, it won't make sense for this one to be the default any more than anything else.

Reply:
Makes sense